documentation/how-to/!how-to-template.md

```diff
@@ -0,0 +1,139 @@
+---
+title: {Guide Title for a Specific Task}
+---
+<!-- template preamble -->
+
+## Template preamble (remove!)
+
+A how-to page shows how to do a single thing, typically by giving a short
+sequence of steps. How-to pages have minimal explanation, but often provide links
+to conceptual topics that provide related background and knowledge.
+
+How-tos should provide the available environments (`test`/`prod`) and endpoints, when applicable.
+
+To write a new how-to page, copy this file. All text in {curly brackets} should be replaced or removed.
+
+For more information on how-to pages, read the [Good Docs Project guide](https://gitlab.com/tgdp/templates/-/blob/main/how-to/guide-how-to.md).
+
+---
+
+<!-- overview -->
+
+This guide explains how to {insert a brief description of the task}.
+
+{Optional: Specify when and why your user might want to perform the task.}
+
+{Optional: Include links/tooltips to relevant concepts when terms are introduced here or in the `steps` section.}
+
+<!-- prerequisites -->
+
+## Before you begin
+
+{Optional: You should be familiar with how to do a [more basic task](/how-to/gwa-install.md) or [concept](/concepts/api-directory.md).}
+
+Before you begin, ensure you:
+
+- [Install gwa CLI](/how-to/gwa-install.md)
+- [Create a Namespace](/resources/gwa-commands.md#namespacecreate)
+- [Complete This](/how-to/create-gateway-service.md)
+- [Do That](/how-to/generate-service-account.md)
+
+<!-- steps -->
+
+## {Sub-task name}
+
+{Optional: Provide a concise description of the purpose of this sub-task. Only include this if the purpose is not clear from the sub-task title.}
+
+{You can use this format to describe steps to complete a sub-task:}
+
+1. {Write the first step here. Use a verb to start.}
+
+  {Optional: Explanatory text}
+
+  {Optional: Code sample or screenshot that helps your users complete this step.}
```

Comment:
is there guidance on when we do/don't want to use screenshots? I know they can be cumbersome to maintain if we make updates to the visuals

Reply:
I added a note on this below. The context the user is in on the page is going to typically be enough.

If we find we have more basic users and they are struggling, we should consider revising the UI and also adding more screenshots for those less technical processes.